setup.cfg

```diff
@@ -31,5 +31,78 @@ select=
     # F402: import 'assert_equal' from line 7 shadowed by loop variable
     W601,
     # W601: .has_key() is deprecated, use 'in'
-    W391
+    W391,
     # W391: blank line at end of file
+    E112,
+    # E112: expected an indented block
+    E113,
+    # E113: unexpected indentation
+    E223,
+    # E223: tab before operator
+    E224,
+    # E224: tab after operator
+    E242,
+    # E242: tab after ‘,’
+    E273,
+    # E273: tab after keyword
+    E274,
+    # E274: tab before keyword
+    E275,
+    # E275: missing whitespace after keyword
+    E304,
+    # E304: blank lines found after function decorator
+    E742,
+    # E742: do not define classes named ‘l’, ‘O’, or ‘I’
+    E743,
+    # E743: do not define functions named ‘l’, ‘O’, or ‘I’
+    E902,
+    # E902: IOError
+    W602,
+    # W602: deprecated form of raising exception
+    W603,
+    # W603: ‘<>’ is deprecated, use ‘!=’
+    W604,
+    # W604: backticks are deprecated, use ‘repr()’
+    W605,
+    # W605: invalid escape sequence ‘x’
+    W606,
+    # W606: ‘async’ and ‘await’ are reserved keywords starting with Python 3.7
+    F404,
+    # F404: future import(s) name after other statements
+    F406,
+    # F406: ‘from module import *’ only allowed at module level
+    F407,
+    # F407: an undefined __future__ feature name was imported
+    F601,
+    # F601: dictionary key name repeated with different values
+    F602,
+    # F602: dictionary key variable name repeated with different values
+    F621,
+    # F621: too many expressions in an assignment with star-unpacking
+    F622,
+    # F622: two or more starred expressions in an assignment (a, *b, *c = d)
+    F631,
+    # F631: assertion test is a tuple, which are always True
+    F701,
+    # F701: a break statement outside of a while or for loop
+    F702,
+    # F702: a continue statement outside of a while or for loop
+    F703,
+    # F703: a continue statement in a finally block in a loop
+    F704,
+    # F704: a yield or yield from statement outside of a function
+    F705,
+    # F705: a return statement with arguments inside a generator
+    F706,
+    # F706: a return statement outside of a function/method
+    F707,
+    # F707: an except: block as not the last exception handler
+    F721,
+    # F721: doctest syntax error
+    F722,
+    # F722: syntax error in forward type annotation
+    F831,
+    # F831: duplicate argument name in function definition
+    F901
+    # F901: raise NotImplemented should be raise NotImplementedError
```

Comment:
we're currently checking for this as E901.  Are there two slightly different versions of this or is one of these a typo?

Reply:
Then I guess I must have made a typo way back when.  Might be worth updating the message attached to E901 up at the top